core/src/main/java/org/geysermc/geyser/level/chunk/GeyserChunkSection.java

```diff
@@ -55,15 +52,17 @@ public void setFullBlock(int x, int y, int z, int layer, int fullBlock) {
     }
 
     public void writeToNetwork(ByteBuf buffer) {
-        buffer.writeByte(CHUNK_SECTION_VERSION);
+        if (this.storage.length != 1) {
+            buffer.writeByte(8);
```

Comment:
Why was this constant removed?

Reply:
Because I'm using v1 and v8, and this is to not confuse that I'm only using v8, could make the code a little bit more clear, because for v1 I'm using this.storage.length when it is 1 ^^'

Reply:
That doesn’t change anything in the one place it’s used though?

Reply:
Nah but then I should more it more clear, like:
`if (this.storage.length == 1) {
    buffer.writeByte(CHUNK_SECTION_VERSION_1);
} else {
    buffer.writeByte(CHUNK_SECTION_VERSION_8);
    buffer.writeByte(this.storage.length);
}
for (BlockStorage blockStorage : this.storage) {
    blockStorage.writeToNetwork(buffer);
}`
just a semantic thing, but ignoring that I'll add it back again